tower-grpc-interop/travis-interop.sh

```diff
@@ -24,4 +24,4 @@ trap 'echo ":; killing test server"; kill ${SERVER_PID};' EXIT
 
 # run the interop test client against the server.
 cargo run -p tower-grpc-interop --bin client -- \
-    --test_case=empty_unary     # this is the only working test case...
\ No newline at end of file
+    --test_case=empty_unary,large_unary
```

Comment:
This LGTM, although I _think_ that if you want to have them on multiple lines, you should be able to say 
```bash
cargo run -p tower-grpc-interop --bin client -- \
  --test_case=empty_unary,\
              large_unary
```